ghp-import

```diff
@@ -72,6 +72,8 @@ def try_rebase(remote, branch):
     cmd = ['git', 'rev-list', '--max-count=1', 'origin/%s' % branch]
     p = sp.Popen(cmd, stdin=sp.PIPE, stdout=sp.PIPE, stderr=sp.PIPE)
     (rev, ignore) = p.communicate()
+    if isinstance(rev, bytes):
+        rev = rev.decode()
```

Comment:
Any reason why you didn't just call dec(rev.strip()) with the dec function defined earlier (it does the type check already)? That works for me. AFAICT L77 should just be:

cmd = ['git', 'update-ref', 'refs/heads/%s' % branch, dec(rev.strip())]